pkg/helpers/tokencmd/request_token.go

```diff
@@ -461,7 +461,7 @@ func transportWithSystemRoots(issuer string, clientConfig *restclient.Config) (h
 		}
 		return systemRootsRT, nil
 	case x509.UnknownAuthorityError, x509.HostnameError, x509.CertificateInvalidError, x509.SystemRootsError,
-		tls.RecordHeaderError, *net.OpError:
```

Comment:
this switch case is checking for specific verification errors. For now, we could just unwrap the error before the switch I guess